client/extensions/woocommerce/app/settings/email/mailchimp/getting-started.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import { localize } from 'i18n-calypso';
+
+class MailChimpGettingStarted extends React.Component {
```

Comment:
This seems like it would be a good candidate for a `PureComponent` since the props are simplistic. Missing `propTypes` declaration too for `onClick`